src/hammer-vlsi/hammer_vlsi/driver.py

```diff
@@ -75,6 +75,9 @@ def __init__(self, options: HammerDriverOptions, extra_project_config: dict = {}
         # Store the run dir.
         self.obj_dir = options.obj_dir  # type: str
 
+        # Also store the options
+        self.options = options
```

Comment:
What's the context for this change?

Reply:
We need to know the actual paths of the input confs, so that they can be passed along to the build tool.